packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx

```diff
@@ -102,7 +99,8 @@ export class ColorPickerGridCellBase extends React.Component<IColorPickerGridCel
    * @returns - Whether the cell's color is white or not.
    */
   private _isWhiteCell(inputColor: string | undefined): boolean {
-    return inputColor!.toLocaleLowerCase() === '#ffffff';
+    const color = getColorFromString(inputColor!);
+    return color!.r === COLOR_VALUES.white[0] && color!.g === COLOR_VALUES.white[1] && color!.b === COLOR_VALUES.white[2];
```

Comment:
The color object returned from the `getColorFromString()` has also the hex value of the color. So maybe better than importing `COLOR_VALUES` you could do the boolean check for `color.hex === 'ffffff'`.